File: 03 - Simple Object Oriented Language Examples/class-example.py
#!/usr/bin/env python

'''

Very simple example of object oriented programming in Python

'''

class Picture():

  def __init__(self, id, title, img):
    self.id = id
    self.title = title
    self.img = img

  def display_values(self):
    print("id:%s, title:%s, img:%s" % (self.id, self.title, self.img))

# initialize and call
example = Picture("1", "title", "file.png")
example.display_values()

File: 03 - Simple Object Oriented Language Examples/class-example.js
<script>

/*

Very simple example of object oriented programming in JavaScript

*/

class Picture {
  constructor(id, title, img) {
    this.id = id;
    this.title = title;
    this.img = img;
  }

  display_values() {
    console.log("id:%s, title:%s, img:%s", this.id, this.title, this.img);
  }
}

const example = new Picture("1", "title", "file.png");
example.display_values();

</script>
